graphene_django/static/graphene_django/graphiql.js

```diff
@@ -61,17 +61,29 @@
   var fetchURL = locationQuery(otherParams);
 
   // Defines a GraphQL fetcher using the fetch API.
-  function httpClient(graphQLParams) {
-    var headers = {
-      Accept: "application/json",
-      "Content-Type": "application/json",
-    };
+  function httpClient(graphQLParams, opts = { headers: {} }) {
+    let headers = opts.headers;
+    // Convert headers to an object.
+    if (typeof headers === 'string') {
+      headers = JSON.parse(opts.headers);
+    }
```

Comment:
I'm not sure where the `string` type comes from, as in `graphiql` definitions, `headers` can only be an object ([source](https://unpkg.com/browse/graphiql@1.0.3/esm/components/GraphiQL.d.ts)):

```ts
export declare type FetcherParams = {
  query: string;
  operationName: string;
  variables?: string;
};
export declare type FetcherOpts = {
  headers?: {
    [key: string]: any;
  };
  shouldPersistHeaders: boolean;
};
export declare type FetcherResult = {
  data: IntrospectionQuery;
} | string | {
  data: any;
};
export declare type Fetcher = (graphQLParams: FetcherParams, opts?: FetcherOpts) => Promise<FetcherResult> | Observable<FetcherResult>;
```